package/MDAnalysis/coordinates/XDR.py

```diff
@@ -116,8 +116,12 @@ def _read_offsets(self, store=False):
                 np.savez(offsets_filename(self.filename),
                          offsets=offsets, size=size, ctime=ctime)
             except Exception as e:
-                warnings.warn("Couldn't save offsets because: {}".format(
-                    e.message))
+                try:
```

Comment:
It seems that some exceptions do not have a message attribute. I was receiving errors about `PermissionError` not having it.